sdcm/tombstone_gc_verification_thread.py

```diff
@@ -0,0 +1,182 @@
+import datetime
+import json
+import logging
+import random
+import threading
+import time
+
+from sdcm import wait
+from sdcm.cluster import BaseScyllaCluster, BaseCluster
+from sdcm.remote import LocalCmdRunner
+from sdcm.sct_events import Severity
+from sdcm.sct_events.database import TombstoneGcVerificationEvent
+from sdcm.utils.decorators import retrying
+
+LOCAL_CMD_RUNNER = LocalCmdRunner()
+ERROR_SUBSTRINGS = ("timed out", "timeout")
+
+
+class NonDeletedTombstonesFound(Exception):
+    pass
+
+
+class NoSstableFound(Exception):
+    pass
+
+
+# pylint: disable=too-many-instance-attributes
+class TombstoneGcVerificationThread:
+
+    PROPAGATION_DELAY = 1  # TODO: update the real value.
```

Comment:
what is the `real value`?

Reply:
It seems like scylla sets it by default to 1 hour:
https://github.com/scylladb/scylladb/blob/5ed1a8217c10e87373a5a38a7afa879212bddd22/tombstone_gc_options.hh#L20

But it's node clear for me why a response time from node would be so long. @asias , can you please confirm this is the case?

I guess we can try Altering the tested table to have a much lower value like 5 minutes. (in Dtest it's 5 seconds).

Reply:
Currently fixed by:
```
# Set tombstone_gc to 'repair' + a propagation_delay_in_seconds of 5 minutes for the tombstone-gc-verification table:
post_prepare_cql_cmds: "ALTER TABLE cqlstress_lwt_example.blogposts with tombstone_gc = {'mode': 'repair', 'propagation_delay_in_seconds':'300'};"
run_tombstone_gc_verification: '{"ks_cf": "cqlstress_lwt_example.blogposts", "interval": 300, "propagation_delay_in_seconds": 300}' # 'ks.cf, interval(sec), propagation_delay_in_seconds(sec)'
```